### PR TITLE
[RFC]autoload sj-app

### PR DESCRIPTION
**Pull request by mattn**

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -8,3 +8,17 @@ const Element = require('./element.js');
...
+  const elems = document.querySelectorAll('[sj-app]');
+  for (let i=0, l=elems.length; i<l; ++i) {
+    const elem = elems[i];
+    const template = document.createElement("sj-template");
+    template.innerHTML = elem.innerHTML;
+    elem.parentNode.replaceChild(template, elem);
+  }
+  customElements.define('sj-template', class extends sj.Element {
+    template() { return this.innerHTML; }
+    initialize() { }
```
Comment (tokuhirom) [inline src/main.js]: You don't need to override this method.

**Comment by tokuhirom** [comment]
looks good to me

**Comment by tokuhirom** [comment]
I wonder if sj calls `window['MyApp'].apply(elem)` if `sj-app="MyApp"` was specified.

**Comment by tokuhirom** [comment]
Disable `{{ }}` processing in sj-app for security reason.

**Comment by mattn** [comment]
See #12 but the issue about {{}} are remaining.

**Pull request closed without merging**